.github/workflows/ci.yaml

```diff
@@ -43,4 +44,7 @@ jobs:
             target/
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
       - name: build all
-        run: make build-all
+        run: |
+          chmod a+x /opt/hostedtoolcache/streamingfast/substreams/v0.0.21/linux-x64/substreams_0.0.21_linux_x86_64/substreams
```

Comment:
any way we can avoid hardcoding this?

Reply:
Don't think so. I have fixed the version for the substreams CLI to v21 so it definitely won't start breaking when they make the next release. And once we see they've made the first release with the fix to allow for proper unpacking of the binary during the installation stage we can revert the code back.

Reply:
gotcha, sounds good